cico_setup.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Output command before executing
+set -x
+
+# Exit on error
+set -e
+
+REPO_NAME="almighty"
+CURRENT_DIR=$(pwd)
+PROJECT_NAME="keycloak"
+DOCKER_IMAGE_CORE=$PROJECT_NAME
+DOCKER_IMAGE_DEPLOY=$PROJECT_NAME-deploy
+
+KEYCLOAK_VERSION="3.0.0.Final"
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+function load_jenkins_vars() {
+  if [ -e "jenkins-env" ]; then
+    cat jenkins-env \
+      | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+      | sed 's/^/export /g' \
+      > ~/.jenkins-env
+    source ~/.jenkins-env
+  fi
+}
+
+function install_deps() {
+  # We need to disable selinux for now, XXX
+  /usr/sbin/setenforce 0
+
+  # Get all the deps in
+  yum -y install \
+    docker \
+    make \
+    git \
+    wget \
+    curl
+
+  wget http://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo -O /etc/yum.repos.d/epel-apache-maven.repo
+  yum -y install apache-maven
+
+  service docker start
+  echo 'CICO: Dependencies installed'
+}
+
+function build() {
+  echo 'CICO: Cloning keycloak source code repo'
+  git clone https://github.com/almighty/keycloak.git --branch move_deployment_out
```

Comment:
90% of time it would be a tag of the latest upstream GA release actually.
And I meant the local var in the script which would be easily to locate during update. But my main concern is actually that we also use this branch/tag name in the cico_run_tests.sh script.
So, the "normal" workflow of updating KC would be:
1. Upstream releases a new version, let's say 4.0.0.
2. We pull 4.0.0 tag from upstream to almighty-core/keycloak
3. We update cico_setup.sh (and cico_run_tests.sh) to clone almighty-core/keycloak --branch 4.0.0.Final (or whatever they name the tag)